profile/README.md

```diff
@@ -1,28 +1,40 @@
 
-- **Nteract:** a 100% Rust Android Automotive & Linux compatible HAL providing secure & safe APIs to communicate to any vehicle. Nteract acts as the security gateway between the non-secure, connected "Android & Linux" world and the in-vehicle highly secure domain. This allows for zero-cost secure integration of HMI and applications 
+Veecle contributes with two keys components to the ecosystem:
 
-- **Veecle Studio & Cloud Services** consist of a set of tools that can be used to deploy, test, monitor and manage our instances. At its core, this is a set of secure APIs that allows interconnecting visual tools for development, deployment, data visualisation, data storage and fleet management
+- [Veecle-pxros](https://github.com/veecle/veecle-pxros) A runtime for writing Rust applications on top of HighTec's safety kernel for the Infineon AURIX™ Platform.
+- [Tricore-probe](https://github.com/veecle/tricore-probe): A tool that facilitates the flashing and debugging of Rust code on AURIX™ chips.
 
-## What is currently published
+### Android Automotive support
 
-- [TC37x-demo](https://github.com/veecle/tc37x-demo): demo code to run a 100% Rust basic CAN driver on AURIX™ TC375 LK
-- [Tricore-probe](https://github.com/veecle/tricore-probe): a Rust probe for AURIX™ (currently works on Windows and Linux)  
+We support [Android Automotive](https://source.android.com/docs/automotive/start/what_automotive): this enables customers to develop safe and efficient interfaces to the vehicle. 
 
-These are demos! For full AURIX™ Rust support package please write us at info@veecle.io
+This is not public (yet). You can find information [here](https://veecle.io/tech/aaos).
 
-## Upcoming
-**Stay tuned! Our first release is coming soon!**
+### Robot Operating System support
 
-We are working towards opening our codebase and first release! In the meantime, if you wish to get access to NoS or any of our product please 
-visit our website or write us at info@veecle.io 
+We integrate to the [Robot Operating System](https://www.ros.org) to allow for easy integration with enabled devices over the [Data Distribution Service](https://www.dds-foundation.org/what-is-dds-3). 
 
-## Learn more
-Are you interested in Veecle? \
-Visit us at www.veecle.io or write us at info@veecle.io
+This is not public (yet). You can find information [here](https://veecle.io/tech/dds).
+
+## About Veecle
+
+Veecle sits between a legacy electrical/electronic (E/E) architecture, and modern UI and Cloud frameworks. Veecle abstracts an E/E architecture and provides unified control over it using a modern API. Veecle allows companies to develop applications using unified APIs to interact with in-vehicle functions. An optional front-end framework is provided through Android Automotive support.
```

Comment:
```suggestion
Veecle
* sits between a legacy electrical/electronic (E/E) architecture, and modern UI and Cloud frameworks.
* abstracts an E/E architecture and provides unified control over it using a modern API.
* allows companies to develop applications using unified APIs to interact with in-vehicle functions. An optional front-end framework is provided through Android Automotive support.
```

Its a lot of Veecle does.

Reply:
It's fine but not really a nice paragraph structure. If possible, this should be changed on the website too.